Triangulation_3/include/CGAL/IO/Triangulation_raw_iostream_3.h

```diff
@@ -0,0 +1,396 @@
+// Copyright (c) 20XX,20XX GeometryFactory
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Ange Clément
+
+
+#ifndef CGAL_TRIANGULATION_RAW_IOSTREAM_3_H
+#define CGAL_TRIANGULATION_RAW_IOSTREAM_3_H
+
```

Comment:
To following naming convention:
```suggestion
#ifndef CGAL_IO_TRIANGULATION_RAW_IOSTREAM_3_H
#define CGAL_IO_TRIANGULATION_RAW_IOSTREAM_3_H

```